### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Michael Markl, Paula Wikidal, Marin Brieger, Roman Buchholz, Elias Keis, Maximilian Ammann
```
Comment (RoAnBu) [inline LICENSE:3]: Find the typo 😆

**Review by RoAnBu — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (Schedulaar) [inline LICENSE:3]:
```suggestion
Copyright (c) 2020 Michael Markl, Paula Wikidal, Marvin Brieger, Roman Buchholz, Elias Keis, Maximilian Ammann
```

**Review by Schedulaar — APPROVED** [review]
I think everybody should approve this!

**Review by elKei24 — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (elKei24) [reply LICENSE:3]: We maybe could also just do something like the following to be safe for the future:
```suggestion
Copyright (c) 2020 The Digitale Ehrenamtskarte contributors
```

**Review by RoAnBu — APPROVED** [review]

**Pull request merged**